util/container/Dockerfile

```diff
@@ -151,3 +166,13 @@ COPY --from=snax-gemm /src/target/snitch_cluster/sw/snax/ /opt/snax-gemm/target/
 COPY --from=snax-gemm /src/sw/math/ /opt/snax-gemm/sw/math/
 COPY --from=snax-gemm /src/sw/deps/riscv-opcodes /opt/snax-gemm/sw/deps/riscv-opcodes
 COPY --from=snax-gemm /src/sw/deps/printf /opt/snax-gemm/sw/deps/printf
+
+COPY --from=snax-alu /src/target/snitch_cluster/bin/snitch_cluster.vlt /opt/snax-alu-rtl/bin/snitch_cluster.vlt
+COPY --from=snax-alu /src/sw/snRuntime /opt/snax-alu/sw/snRuntime
+COPY --from=snax-alu /src/target/snitch_cluster/sw/runtime/rtl /opt/snax-alu/target/snitch_cluster/sw/runtime/rtl
+COPY --from=snax-alu /src/target/snitch_cluster/sw/runtime/rtl-generic /opt/snax-alu/target/snitch_cluster/sw/runtime/rtl-generic
+COPY --from=snax-alu /src/target/snitch_cluster/sw/runtime/common /opt/snax-alu/target/snitch_cluster/sw/runtime/common
+COPY --from=snax-alu /src/target/snitch_cluster/sw/snax/ /opt/snax-alu/target/snitch_cluster/sw/snax
+COPY --from=snax-alu /src/sw/math/ /opt/snax-alu/sw/math/
+COPY --from=snax-alu /src/sw/deps/riscv-opcodes /opt/snax-alu/sw/deps/riscv-opcodes
+COPY --from=snax-alu /src/sw/deps/printf /opt/snax-alu/sw/deps/printf
```

Comment:
maybe you could also do this:
https://stackoverflow.com/questions/30256386/how-to-copy-multiple-files-in-one-layer-using-a-dockerfile

Reply:
It doesn't seem straightforward with all the nested folders going to different destinations. If ok I'll ignore this as I wanna move fast today, but I'll look into this maybe later (maybe we can just remove all the copies as im not sure we need all the software duplicated in the docker)